### PR TITLE
[Fix] State meta description strings.

### LINKED ISSUE #1921: [Language issue] Mortality Rate on state pages say "0 have unfortunately passed away per 100 confirmed"
### The Issue
image

Please correct this by providing alternative statement to translate when there's no death cases.

### Suggested alternative wording in English
`For every 100 confirmed cases, fortunately no one has died from the virus.`

### PR DESCRIPTION
**Pull request by anshulk**
**Description of PR**

Added default description strings when statistic's value is 0
* Currently, there are no active cases in this state.
* Unfortunately, there are no recoveries in this state yet.
* Fortunately, no one has passed away from the virus in this state.
* There has been no growth in the number of cases in last 7 days.
* No tests have been conducted in this state yet.

**Relevant Issues**  
Fixes #1921 

**Checklist**

- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop
- [x] Tested on phone

**Screenshots**

Add relevant screenshots here

### CODE REVIEW

File: src/components/statemeta.js [hunk 1]
```diff
@@ -129,9 +141,13 @@ function StateMeta({
...
           date={`${sevenDayBeforeDate} - ${previousDayDate}`}
-          description={`In the last one week, the number of new infections has grown by an average of ${Math.round(
-            growthRate / 7
-          )}% every day.`}
+          description={
+            growthRate != 0
+              ? `In the last one week, the number of new infections has grown by an average of ${Math.round(
+                  growthRate / 7
+                )}% every day.`
+              : `There has been no growth in the number of cases in last 7 days.`
```
Comment (bee-rickey) [inline src/components/statemeta.js]: Lets keep this in sync with the other message? Replace 7 days with last week, cases with infections?

**Comment by bee-rickey** [comment]
@shuklaayush @jeremyphilemon I think we should merge this. This is a good change to the messages being displayed.

File: src/components/statemeta.js (hunk 1 above)
Comment (anshulk) [reply src/components/statemeta.js]: Sure, added a new commit with the change.

**Comment by anukaal** [comment]
Yes , it will be good change in the messages being displayed

**Pull request merged**